Pipfile

```diff
@@ -8,6 +8,7 @@ requests = "*"
 thoth-common = "*"
 thoth-solver = "*"
 thoth-storages = "*"
+thoth-messaging = "==0.7.13"
```

Comment:
the new message `CVEProvidedMessage` won't be available in that version

Reply:
let's leave it as `*` for now, https://github.com/thoth-station/cve-update-job/pull/374/commits/519c78651affb32e23a2e3406c8acdc45818c219